assembly/src/procedures/mod.rs

```diff
@@ -204,8 +203,8 @@ impl ProcedureId {
         L: AsRef<str>,
     {
         let mut digest = [0u8; Self::SIZE];
-        let hash = Blake3_256::<Felt>::hash(path.as_ref().as_bytes());
-        digest.copy_from_slice(&hash.as_bytes()[..Self::SIZE]);
+        let hash = Blake3_256::hash(path.as_ref().as_bytes());
+        digest.copy_from_slice(&(*hash)[..Self::SIZE]);
```

Comment:
Why not use `Blake3_160` here?

Reply:
We could use `192` as we need 24 bytes. Updated!